collect_sources.sh

```diff
@@ -82,7 +82,7 @@ wget -O wiringpi_$version.tar.gz \
 	https://github.com/WiringPi/WiringPi/archive/final_official_$version.tar.gz
 
 # fetch RevolutionPi sources
-wget -O linux.tar.gz https://github.com/RevolutionPi/linux/archive/revpi-4.9.tar.gz
+wget -O linux.tar.gz https://github.com/RevolutionPi/linux/archive/refs/tags/raspberrypi-kernel_9.20210326-4.19.95+revpi1.tar.gz
```

Comment:
This is useless as well. After a kernel Package upgrade this is wrong again!

Reply:
should be fetch the correct version from apt-cache policy raspberrypi-kernel ?

Reply:
We need to fetch the kernel source matching the installed kernel package! The version can't be hardcoded in this script

Reply:
So match the tag  (e.g. raspberrypi-kernel_9.20210326-4.19.95+revpi1) with the output of "apt show raspberrypi-kernel | grep Version: " should be a solution, right?

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Version: 9.20210316-4.19.95+revpi1

Reply:
or maybe we can provide the source package of raspberrypi-kernel